cms/envs/devstack.py

```diff
@@ -48,6 +48,9 @@
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
 ]
 
+# Load development webpack donfiguration
+WEBPACK_CONFIG_PATH = 'webpack.dev.config.js'
```

Comment:
This now matches the `pavelib/assets.py` behavior of "development in devstack, else production"